src/OpenSpaceToolkit/Astrodynamics/GuidanceLaw/QLaw.cpp

```diff
@@ -636,6 +676,51 @@ Vector5d QLaw::computeDeltaCOE(const Vector5d& aCOEVector) const
     };
 }
 
+Tuple<double, double> QLaw::computeEffectivity(
+    const Vector6d& aCOEVector, const Vector3d& currentThrustDirection, const Vector5d& dQ_dOE
```

Comment:
Maybe just add comment with paper and section? 🙏

Reply:
💯 this is a great idea. I'll add some details as this was not trivial for me to find as well.